bower.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "hopscotch",
+  "version": "0.2.3",
+  "authors": [],
+  "description": "A framework to make it easy for developers to add product tours to their pages.",
+  "main": [
+    "./dist/js/hopscotch.js",
+    "./dist/css/hopscotch.css"
+  ],
+  "keywords": [
+    "tour",
+    "linkedin"
+  ],
+  "license": "Apache License",
+  "homepage": "https://github.com/linkedin/hopscotch",
+  "ignore": [
+    "**/.*",
```

Comment:
No, it will ignore hidden files — started with a dot. Actually those lines were generated by bower itself. 

Also I'm wondering if we really need archieve folder in production. I've pushed an additional commit to add them to ignore list.